packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/_layout.tsx

```diff
@@ -4,23 +4,30 @@ import {
   modalFirstScreenOptions,
   modalStackNavigatorScreenOptions
 } from 'common/consts/screenOptions'
+import { SeedlessMnemonicExportProvider } from 'features/accountSettings/context/SeedlessMnemonicExportProvider'
 
 export default function AccountSettingsLayout(): JSX.Element {
   return (
-    <Stack screenOptions={modalStackNavigatorScreenOptions}>
-      <Stack.Screen name="index" options={modalFirstScreenOptions} />
-      <Stack.Screen name="account" />
-      <Stack.Screen name="selectCurrency" />
-      <Stack.Screen name="selectAppearance" />
-      <Stack.Screen name="securityAndPrivacy" />
-      <Stack.Screen name="connectedSites" />
-      <Stack.Screen name="recoveryMethods" />
-      <Stack.Screen name="showRecoveryPhrase" />
-      <Stack.Screen name="changePin" />
-      <Stack.Screen name="verifyChangePin" />
-      <Stack.Screen name="notificationPreferences" />
-      <Stack.Screen name="manageAccounts" />
-      <Stack.Screen name="addressBook" />
-    </Stack>
+    <SeedlessMnemonicExportProvider>
+      <Stack screenOptions={modalStackNavigatorScreenOptions}>
+        <Stack.Screen name="index" options={modalFirstScreenOptions} />
+        <Stack.Screen name="account" />
+        <Stack.Screen name="selectCurrency" />
+        <Stack.Screen name="selectAppearance" />
+        <Stack.Screen name="securityAndPrivacy" />
+        <Stack.Screen name="connectedSites" />
+        <Stack.Screen name="showRecoveryPhrase" />
```

Comment:
can we move this SeedlessMnemonicExportProvider closer to where it is needed? there are quite a few screens here that don't depend on it